### DISCUSSION TITLE
How to get Check Box as First Column

### DISCUSSION THREAD

**Discussion by gopusrikanth04** [TanStack/table#3601] [Q&A]
Hey,

I am trying to apply grouping and row selection .  I am getting the check box after the grouped columns. 
as shown below

![image](https://user-images.githubusercontent.com/47174113/147661297-8684d007-e190-4a79-8d7e-71389e9c4fb1.png)

I want it as first column. How to fix this problem 

I render the Table like below code sample

https://codesandbox.io/s/github/ggascoigne/react-table-example?file=/src/Table/Table.tsx:657-668

Thank You

**Comment by Chalkzhu** [comment]
You can consider adding position: sticky; left: 0;
